common/network-common/src/main/java/org/apache/spark/network/crypto/TransportCipher.java

```diff
@@ -44,7 +44,8 @@
   @VisibleForTesting
   static final String ENCRYPTION_HANDLER_NAME = "TransportEncryption";
   private static final String DECRYPTION_HANDLER_NAME = "TransportDecryption";
-  private static final int STREAM_BUFFER_SIZE = 1024 * 32;
+  @VisibleForTesting
```

Comment:
I think `VisibleForTesting` annotation used for a member with modifier `final` is OK.